thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/template.hbs

```diff
@@ -1,55 +1,307 @@
+            {{/if}}
           </li>
-          <li class="te-horizontal-cards__card-text">
-            {{#if card.projected}}
-              Projected: <strong>{{card.projected}}</strong>
+
+          <li class="te-horizontal-cards__card-value">
+            <div class="te-horizontal-cards__val-group">
+              {{#if isTunePreviewActive}}
+                <div class="te-horizontal-cards__card-subt">Current</div>
+              {{/if}}
+              <div class="te-horizontal-cards__card-number">
+                {{card.value}}
+                {{#if (and card.units (not-eq card.value "N/A"))}}
+                  <span class="te-horizontal-cards__card-units">{{card.units}}</span>
+                {{/if}}
+              </div>
+            </div>
+            {{#if isTunePreviewActive}}
+              <div class="te-horizontal-cards__val-group">
+                <i class="te-horizontal-cards__val-glyph--middle glyphicon glyphicon-menu-right"></i>
+              </div>
+              <div class="te-horizontal-cards__val-group">
+                <div class="te-horizontal-cards__card-subt">New</div>
+                <div class="te-horizontal-cards__card-number">
+                  {{#if (not-eq card.value "N/A")}}
+                    <i class="te-horizontal-cards__val-glyph--right glyphicon glyphicon-triangle-{{if (gt card.projected card.value) "top" "bottom"}}"></i>
+                  {{/if}}
+                  {{card.projected}}
+                  {{#if (and card.units (not-eq card.projected "N/A"))}}
+                    <span class="te-horizontal-cards__card-units">{{card.units}}</span>
+                  {{/if}}
+                </div>
+              </div>
             {{/if}}
           </li>
+
         </ul>
       {{/each}}
+
     </div>
+    {{#if isTunePreviewActive}}
+      <div class="te-form__cta-button">
+        {{bs-button
+          defaultText="Save tuning"
+          type="primary"
+          onClick=(action "onSubmitTuning")
+          buttonType="submit"
+          class="te-button te-button--submit"
+        }}
+      </div>
+    {{/if}}
+
+    {{#if isTuneSaveSuccess}}
+      {{#bs-alert type="success" class="te-form__banner te-form__banner--success"}}
+        <div class="te-form__banner-title">Success:</div> Filters modified.
+      {{/bs-alert}}
+    {{/if}}
+
+    {{#if isTuneSaveFailure}}
+      {{#bs-alert type="danger" class="te-form__banner te-form__banner--failure"}}
+        <strong>Error:</strong> {{failureMessage}}
+      {{/bs-alert}}
+    {{/if}}
+
   </div>
 
+  {{#if isTunePreviewActive}}
+    <div class="te-horizontal-cards te-content-block">
+      <h4 class="te-alert-page__subtitle">Anomalies removed with settings</h4>
+      {{#if diffedAnomalies}}
+        <p>The following previously sent anomalies <strong>would not have sent</strong> under the new settings.</p>
+      {{else}}
+        <p>These settings would result in <strong>NO reduction</strong> of sent anomalies</p>
```

Comment:
Same as above. We can wrap the bolded text in a `span` tag and assign a styling to it.

Reply:
done